api/migrations/20210427172703-legislationToArray.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+const RecordTypeEnum = require('../src/utils/constants/misc');
+
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function(options, seedLink) {};
+
+exports.up = async function(db) {
+  console.log('**** Transfer legislation field on all records to array ****');
+
+  const mClient = await db.connection.connect(db.connectionString, {
+    native_parser: true
+  });
+
+  try {
+    const nrpti = await mClient.collection('nrpti');
+    const redacted_record_subset = await mClient.collection('redacted_record_subset');
+
+
+    const recordSchemas = [
+      ...RecordTypeEnum.MASTER_SCHEMA_NAMES,
+      ...RecordTypeEnum.LNG_SCHEMA_NAMES,
+      ...RecordTypeEnum.NRCED_SCHEMA_NAMES,
+      ...RecordTypeEnum.BCMI_SCHEMA_NAMES
+    ];
+
+
+    const allRecordsNrpti = await nrpti
+    .find({
+      _schemaName: { $in: recordSchemas },
+    })
+    .toArray();
+
+    const allRecordsRRS = await redacted_record_subset
+    .find({
+      _schemaName: { $in: recordSchemas },
+    })
+    .toArray();
+
+
+    let numUpdatedNrpti = 0;
+    let numUpdatedRRS = 0;
+
+
+    const promisesNrpti = allRecordsNrpti.map(async (record) => {
+
+      if ( record.legislation ) {
+
+        let newLegislation = [record.legislation];
+
```

Comment:
There's a bit too much whitespace rows here, no more than 1 space between code and I suggest this:

```const promisesNrpti = allRecordsNrpti.map(async (record) => {
      if ( record.legislation ) {
        let newLegislation = [record.legislation];
```